### ISSUE TITLE
Errors with open dotnet malware

### ISSUE THREAD

**Issue by dukebarman** [radare/radare2#3522]
```
$ file samples/malware
samples/malware: PE32 executable (GUI) Intel 80386 Mono/.Net assembly, for MS Windows
$ r2 samples/malware
Error: check (String wType)
Error: bad parsing String
Error: bad parsing StringTable
Error: bad parsing (VS_VERSIONINFO StringFileInfo)
 -- Log On. Hack In. Go Anywhere. Get Everything.
[0x0042494e]>
```

File: https://dl.dropboxusercontent.com/u/10688770/net_sample.zip
Pass: infected

https://www.virustotal.com/en/file/e79c5efac2d92268a6607ab9b0ea35720bd10dee73689f45ecdb5734d1b14628/analysis/1445347730/

**Comment by Maijin** [comment]
Those are just Warnings

**Comment by dukebarman** [comment]
Maybe change texts for this?

**Comment by radare** [comment]
Fixed in 5fece33093600974f4361f258adcf3d1c9a6f9bf
